### ISSUE TITLE
Parsing json from ' multipart/form-data' request

### ISSUE THREAD

**Issue by ArVan** [balderdashy/skipper#19]
Hi all.
I noticed that the body parser is missing one big feature. 

See, there are cases when request has `multipart/form-data` content type, and sends `json` parameter in  body. I know that  **skipper** checks content type, and if it is a multipart request, it does not even try to parse the body as json. 

I managed somehow to make this check pass for my special case, but then comes another issue.

If I pass the body to `jsonParser`, it fails because the raw body contains not only json, but also other data when sent as `multipart/form-data`.

Here is an example body:

    ------------------------------f2c58ff7f7c8
    Content-Disposition: form-data; name="json"

    {"event": 'some data'}
    ------------------------------f2c58ff7f7c8--

 There are some services like **HelloSign** that are performing callback requests with  `multipart/form-data` type and `json` body. So this is a very important feature. (currently my project is stuck because of this missing feature :((( )

**Comment by sailsbot** [comment]
Thanks for posting, @ArVan.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
